src/components/stacked-organism.sass

```diff
@@ -13,14 +13,16 @@
   &:hover .inspect-stack
     opacity: .75
 
+  *
+    pointer-events: none
+
```

Comment:
I didn't know about this wildcard syntax.  Neat.

Reply:
That's actually pure css (minus the sass nesting).